chapters/automated-testing/integration-testing.md

```diff
@@ -34,16 +34,17 @@ AspNetCoreTodo/
 
 > If you prefer, you can keep your unit tests and integration tests in the same project. For large projects, it's common to split them up so it's easy to run them separately.
 
-Since the test project will use the classes defined in your main project, you'll need to add a reference to the main project:
+Since the test project will use the classes defined in your main project, you'll need to add a reference to the main project. `cd` to the newly-created project and type:
 
 ```
-dotnet add reference ../AspNetCoreTodo/AspNetCoreTodo.csproj
+dotnet add reference ..\AspNetCoreTodo\AspNetCoreTodo.csproj
```

Comment:
This depends on the OS (Windows vs. linux) - lets keep that consistent across the whole project

Reply:
Fair point.